recipes/devel/git.yaml

```
@@ -32,7 +40,9 @@ buildScript: |
                 --prefix=/usr \
                 --with-curl=${BOB_DEP_PATHS[net::curl-dev]}/usr \
                 --without-python \
-                --without-expat
+                --without-expat \
+                --without-iconv \
+                ${EXTRA:+${EXTRA[@]}}
```

Without double quotes the entries in the array are subject to word splitting. I think this should be prevented:

```suggestion
                ${EXTRA:+"${EXTRA[@]}"}
```